.github/workflows/ci.yml

```diff
@@ -43,51 +42,12 @@ env:
 
 jobs:
 
-  changes:
-    runs-on: ubuntu-latest
-    timeout-minutes: 30
-    # Required permissions
-    permissions:
-      pull-requests: read
-    # Set job outputs to values from filter step
-    outputs:
-      pkgdb: ${{ steps.filter.outputs.pkgdb }}
-    steps:
-    # For pull requests it's not necessary to checkout the code
-    - uses: "actions/checkout@v4"
-
-    - name: "Setup upterm session"
-      if: ${{ (true == inputs.enableUpterm) && (inputs.breakOnJob == github.job) }}
-      uses: owenthereal/action-upterm@v1
-      with:
-        limit-access-to-actor: true
-        limit-access-to-users: ${{ inputs.allowedUptermUsers }}
-        wait-timeout-minutes: 15
-
-    - uses: dorny/paths-filter@v3
-      id: filter
-      with:
-        base: ${{ github.event.merge_group.base_ref || 'main' }}
-        ref: ${{ github.event.merge_group.head_ref || github.ref }}
-        filters: |
-          pkgdb:
-            - ".github/workflows/ci.yml"
-            - "pkgdb/**"
-            - "pkgs/**"
-            - "Justfile"
-            - "flake.nix"
-            - "flake.lock"
-            - "shells/**"
-
-  pkgdb-dev:
-    name: "Pkgdb"
+
+  nix-plugins-dev:
+    name: "Nix Plugins"
     runs-on: ${{ matrix.os }}
     timeout-minutes: 60
 
-    if: ${{ github.event_name == 'workflow_dispatch' || needs.changes.outputs.pkgdb == 'true' }}
```

Comment:
followup: the job still took 5 minutes so it seems like it might still be worth skipping https://github.com/flox/flox/actions/runs/12671418376/job/35313128766?pr=2563

Reply:
1m45s of that was updating the cache, so presumably that and build will be quicker next time?

It might not even make sense to have this job separate from "Nix build" now, come to think of it.

Reply:
I guess you mean the dev builds, but i tend to agree
the "Build" step also took 1m1s out of 1m7s downloading and setting up the dev shell and we're only running two tests here anyway.
OTOH those jobs run in parallel to everything else, so they should not influence our overall test runtime.

Reply:
Do we want to add the skipping back as followup?